### PR TITLE
Update README.md to Express 4

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +80,9 @@ var express = require('express'),
     }
   });
 
-app.use( express.json() );
-app.use( express.urlencoded() );
-app.use( express.methodOverride() );
+app.use( bodyParser.json() );
+app.use( bodyParser.urlencoded() );
```
Comment (ericelliott) [inline README.md]: This will throw an error. Please change to:

```js
app.use( bodyParser.urlencoded({extended: true}) );
```

**Comment by gergelyke** [comment]
just updated

**Comment by ericelliott** [comment]
There's just one other issue: The source code isn't actually compatible with the 4.x line, yet. We also need to update package.json (adding a couple missing dependencies while we're at it). That would look something like this:

```javascript
{
  "name": "siren-resource",
  "version": "0.0.5",
  "description": "Resourceful routing with siren+json hypermedia for Express.",
  "main": "siren-resource.js",
  "dependencies": {
    "cuid": "^1.2.4",
    "dotty": "0.0.2",
    "express": "^4.10.7",
    "mout": "^0.7.1",
    "stampit": "^1.1.0"
  },
  "devDependencies": {
    "body-parser": "^1.10.1",
    "grunt": "^0.4.1",
    "grunt-contrib-jshint": "^0.7.0",
    "method-override": "^2.3.1"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git@github.com:ericelliott/siren-resource.git"
  },
  "keywords": [
    "siren",
    "json",
    "resourceful",
    "express",
    "resource",
    "router",
    "hypermedia",
    "RESTful",
    "REST"
  ],
  "author": "Eric Elliott",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/dilvie/siren-resource/issues"
  }
}
```

**Comment by gergelyke** [comment]
Updated - also deleted express from the deps, as it is not needed

**Comment by gergelyke** [comment]
okay - addad express to devDeps!

**Comment by ericelliott** [comment]
:+1:  Thanks!

Now we just need some unit tests... =)

**Comment by ericelliott** [comment]
I've added you as a collaborator on the project. Please continue to use the pull request process to get your changes merged. I'll start doing the same.

**Pull request merged**